api.bs

```diff
@@ -1377,12 +1377,11 @@ the {{PrivateAttributionImpressionOptions}} dictionary passed to
     Value of <a dict-member for=PrivateAttributionImpressionOptions>histogramIndex</a>,
     a non-negative [=structured header/integer=]. This key is required.
   </dd>
-  <dt><code>filter-data</code></dt>
+  <dt><code>match-value</code></dt>
   <dd>
-    Value of <a dict-member for=PrivateAttributionImpressionOptions>filterData</a>,
-    a non-negative [=structured header/integer=] or an [=structured header/inner list=]
-    of non-negative integers. This key is optional.
-    If not supplied, the empty [=set=] is saved for [=impression/Filter Data=].
+    Value of <a dict-member for=PrivateAttributionImpressionOptions>matchValue</a>,
+    a non-negative [=structured header/integer=]. This key is optional.
+    If not supplied, a value of 0 is saved for [=impression/Match Value=].
```

Comment:
Just flagging that this removes the inner list syntax option here, in line with the decision to allow just one for `saveImpression`